src/media/js/addon/components/addon.js

```diff
@@ -97,6 +109,8 @@ export class Addon extends React.Component {
           <dd>{this.props.version}</dd>
         </dl>
 
+        {this.renderDescription()}
```

Comment:
Definition lists suck in JSX, since you can only have a single root element. That pattern is nice though.